demisto_sdk/commands/common/tools.py

```diff
@@ -3665,3 +3665,36 @@ def parse_marketplace_kwargs(kwargs: Dict[str, Any]) -> MarketplaceVersions:
         "neither marketplace nor is_xsiam provided, using default marketplace=XSOAR"
     )
     return MarketplaceVersions.XSOAR  # default
+
+
+def get_api_module_dependencies_from_graph(changed_api_modules: set, graph) -> List:
```

Comment:
type hints

Reply:
when adding the type hint of the graph Im getting import circular issue